lib/diagnostic.rb

```diff
@@ -13,7 +13,16 @@
 # and location should only be writable.
 
 ##
-# your code here
+class Person
+  attr_reader :age
+  attr_accessor :name
+  attr_writer :location
+  def initialize
+    @name = name
+    @age = age
+    @location = location
+  end
```

Comment:
you forgot to pass in parameters here e.g., `def initialize (name, age, location)`